packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -227,8 +227,7 @@ export default mixins(
 		data () {
 			return {
 				aboutDialogVisible: false,
-				// @ts-ignore
-				basePath: this.$store.getters.getBaseUrl,
+				publicPath: process.env.VUE_APP_PUBLIC_PATH || '/',
```

Comment:
can you please move this to `constants.ts` file to have more separation of concerns?

Reply:
Yes, thank you for your suggestion. I'll do it.